website/docs/r/logs_pipeline.html.markdown

```diff
@@ -0,0 +1,254 @@
+* service_remapper
+  * `name` - (Optional) Name of the processor
+  * `is_enabled` - (Optional, default = false) If the processor is enabled or not.
+  * `sources` - (Required) List of source attributes.
+* status_remapper
+  * `name` - (Optional) Name of the processor
+  * `is_enabled` - (Optional, default = false) If the processor is enabled or not.
+  * `sources` - (Required) List of source attributes.
+* trace_id_remapper
+  * `name` - (Optional) Name of the processor
+  * `is_enabled` - (Optional, default = false) If the processor is enabled or not.
+  * `sources` - (Required) List of source attributes.
+* url_parser
+  * `name` - (Optional) Name of the processor
+  * `is_enabled` - (Optional, default = false) If the processor is enabled or not.
+  * `sources` - (Required) List of source attributes. 
+  * `target` - (Required) Name of the parent attribute that contains all the extracted details from the `sources`.
+  * `normalize_ending_slashes` - (Optional, default = false) If needs to normalize the ending slashes or not.
+* user_agent_parser
+  * `name` - (Optional) Name of the processor
+  * `is_enabled` - (Optional, default = false) If the processor is enabled or not.
+  * `sources` - (Required) List of source attributes.
+  * `target` - (Required) Name of the parent attribute that contains all the extracted details from the sources.
+  * `is_encoded` - (Optional, default = false) If the source attribute is url encoded or not.
+
+## Attributes Reference
+
+* `name` - (Required) The name attribute in resource `datadog_logs_pipelineorder` needs to be unique. It's better to set to the same value as the resource `NAME`. 
+There is no related field can be found in  [Logs Pipeline API](https://docs.datadoghq.com/api/?lang=python#get-pipeline-order).
+* `pipelines` - (Required) The pipeline ids list. The order of pipeline ids in this attribute defines the overall pipelines order for Logs.
+
+**Note:** The [read-only pipelines](https://docs.datadoghq.com/logs/processing/pipelines/#integration-pipelines) also need to be included in the `pipelines`
+list in resource `datadog_logs_pipelineorder`. You can update the order of these pipelines, but you can't modify the **read-only** pipelines 
+through this provider. 
+
+## Import
+
+For the pipelines that are already created (including the **read-only** ones), you can include them into terraform by `import` operation.
+At the moment of writing this documentation, terraform requires to explicitly create resources that match the existing pipelines to be able to
+import them. You will need to do ```terraform import <resource.name> <pipelineID>``` for each existing pipeline.
+
+## Important Notes
+
+Each `datadog_logs_pipeline` resource defines a complete pipeline. The order of pipelines are maintained in resource `datadog_logs_pipelineorder`.
+There should be just one `datadog_logs_pipelineorder` resource. When creating a new pipeline, you need to **explicitly** add this pipeline to 
```

Comment:
I'd rephrase this to "There must be at most one", because otherwise Terraform would demonstrate all kinds of weird behavior.